x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -42,6 +42,7 @@ import {
 import { getCurrentIsolationRequestState } from '../store/selectors';
 import { licenseService } from '../../../../common/hooks/use_license';
 import { FleetActionGenerator } from '../../../../../common/endpoint/data_generators/fleet_action_generator';
+import { APP_PATH, MANAGEMENT_PATH } from '../../../../../common/constants';
```

Comment:
That's perfect! One last thing (shouldn't block this pr being merged). This tests are still using the hardcoded value for `/administration`:
```
x-pack/plugins/security_solution/public/management/pages/index.test.tsx
x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/middleware.test.ts
x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx
x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/trusted_apps_list/index.test.tsx
x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/trusted_apps_grid/index.test.tsx
x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/reducer.test.ts
```

Reply:
Ok, I will replace them in following PRs, we would like to merge this one today